doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+SYSPARM('text') | M | none | Define text string which can be accessed by mz390 global macro variable &SYSPARM.
+SYSPCH(dir) | M | pgm dir | Define directory for PCH output files from PUNCH.  This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).  Note PUNCH output is in ASCII source format and is not directed to the OBJ file.  These files may be used as linker input commands via SYSLKD(file) option.  Set to SYSPCH(.) for current directory when no path is specified on DSNAME parm for PUNCH.
+SYSPRN(dir) | A | pgm dir | Define directory for assembler listing PRN output files.  This option may also override file name and suffix.  This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSTERM(file) | MALE | pgm.ERR | Define alternate file for all start/end messages plus any error messages, plus related source file statistics for each file containing errors.  The default is the program name with ERR suffix.
+SYSTRC(dir) | MALE | pgm dir | Define directory for output TRACE TR? files
+TEST | E | NO | Start execution with interactive TEST mode active which prompts for commands such as T opcode or G opcode to trace or execution to the next occurrence of opcode name.
+TEST(ddname) | E | NO | Define input command file to run TEST commands.  `ddname` is an environment variable set to the name of the command file. See regression test rt\test\TESTTST1.MLC for example.
+THREAD | A | NO | Assign continuing CSECT addresses for multiple CSECT's assembled in the same module to help identify location of instruction and data labels.
+TIME or TIME(seconds) | MALE | YES | Limit execution time of each program to the number of seconds specified.  The default is 15 seconds. This option is turned off if NOTIMING specified.  Use NOTIME to allow tasks such as SOA servers like z390CICS to run continuously while still supporting application use of timing functions.
+TIMING | MALE | YES | Calculate elapsed time and instructions per second statistics for display on START/END messages and STA statistics. tTIMING also displays current z390 and J2SE version on start message and memory usage on ended message.  Note NOTIMING is used in regression tests to force generated files to be identical by using fixed data/time stamp using GregorianCalendar(2005,0,2,22,33,44).  NOTIMING suppresses versions on start message and memory usage on ended message.  NOTIMING also sets NOTIME which prevents application from using time functions..
+TRACE or TRACE(AEGILMPQTV) | E | NO | Turn on ez390 execution trace generation on TRE file and turn off CON.   TRACE will display every instruction executed along with address and value of each operand.  Any combination of the following trace options can be set using the TRACE(...) option.  TRACE(*) will also set TRACEALL.  Note the &SYSTRACE global SETC variable can be used to turn any trace options on or off during execution.
```

Comment:
Remove extra space between first and second sentence. "... CON.   TRACE ..." -> "... CON.  TRACE ...".

Reply:
Fixed 40c73aa